test/setup.mjs

```diff
@@ -1,12 +1,4 @@
-import { LocalTableland } from "../dist/esm/main.js";
+import chai from "chai";
+import chaiAsPromised from "chai-as-promised";
 
-const lt = new LocalTableland({ silent: true });
```

Comment:
Only the end to end tests need an instance running now, so I move instantiation out of the global setup.